presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -881,10 +881,19 @@ private int getHighestQueryPriority()
             if (queuedQueries.isEmpty()) {
                 return 0;
             }
-            return getQueryPriority(queuedQueries.peek().getSession());
+            return getQueryPriority(queuedQueries.peek());
         }
     }
 
+    private int getQueryPriority(ManagedQueryExecution query)
+    {
+        int priority = SystemSessionProperties.getQueryPriority(query.getSession());
```

Comment:
Is it not static imported because of a naming conflict with the enclosing method?

Reply:
That is correct